gen/Project.toml

```diff
@@ -0,0 +1,6 @@
+name = "MPIgen"
+uuid = "01aa870b-fa19-4f9c-adda-70d3c70d3e08"
+
+[deps]
+Clang = "40e3b903-d033-50b4-a0cc-940c62c95e31"
+MPICH_jll = "7cb0a576-ebde-5e09-9194-50597f1243b4"
```

Comment:
Would also be useful to add MPIPreferences (with compat) and the local preference file, to ensure this works correctly?

Reply:
Yes, added.